src/video-player.js

```diff
@@ -214,24 +224,53 @@ class VideoPlayer extends Utils.mixin(Eventable) {
       }
     };
 
-    const _options = options.playerOptions;
-    const _vjs_options = options.videojsOptions;
+    let _playlistWidget;
 
-    console.log(_options, _vjs_options);
+    const initPlaylistWidget = () => {
+      this.videojs.on('playlistcreated', () => {
+        const plwOptions = _options.playlistWidget;
 
+        if (isObj(plwOptions)) {
+          if (_options.fluid) {
+            plwOptions.fluid = true;
```

Comment:
Need to handle a case where we changed `fluid` from `true` to `false` after player creation.